packages/ui/cypress/tests/watched-accounts.cy.ts

```diff
@@ -58,4 +63,146 @@ describe('Watched Accounts', () => {
     settingsPage.accountContainer().should('have.length', 0)
     settingsPage.addButton().should('be.disabled')
   })
+
+  it('can see the provided name displayed for a watched multisig', () => {
+    cy.visit(settingsPageWatchAccountUrl)
+    addWatchAccount(
+      watchMultisigs['multisig-without-pure'].address,
+      watchMultisigs['multisig-without-pure'].name
+    )
+    // ensure the multisig name is displayed in the settings account container
+    settingsPage.accountContainer().within(() => {
+      accountDisplay
+        .nameLabel()
+        .should('be.visible')
+        .should('have.text', watchMultisigs['multisig-without-pure'].name)
+    })
+    // ensure the name is included in the selectable drop-down option
+    topMenuItems.multiproxySelector().should('be.visible').first().click()
+    topMenuItems.multiproxySelectorOption().within(() => {
+      accountDisplay.nameLabel().should('have.text', watchMultisigs['multisig-without-pure'].name)
+    })
+    // ensure the name is displayed in the home page header
+    topMenuItems.homeButton().click()
+    multisigPage.accountHeader().within(() => {
+      accountDisplay.nameLabel().should('have.text', watchMultisigs['multisig-without-pure'].name)
+    })
+  })
+
+  it('can see the provided name displayed for a watched pure', () => {
+    cy.visit(settingsPageWatchAccountUrl)
+    addWatchAccount(
+      watchMultisigs['multisig-with-pure'].pureAddress,
+      watchMultisigs['multisig-with-pure'].name
+    )
+    // ensure the multisig name is displayed in the settings account container
+    settingsPage.accountContainer().within(() => {
+      accountDisplay
+        .nameLabel()
+        .should('be.visible')
+        .should('have.text', watchMultisigs['multisig-with-pure'].name)
+    })
+    // ensure the name is included in the selectable drop-down option
+    topMenuItems.multiproxySelector().should('be.visible').first().click()
+    topMenuItems.multiproxySelectorOption().within(() => {
+      accountDisplay.nameLabel().should('have.text', watchMultisigs['multisig-with-pure'].name)
+    })
+    // navigate to the multisig page and ensure the name is included in the home page header
+    topMenuItems.homeButton().click()
+    multisigPage.accountHeader().within(() => {
+      accountDisplay.nameLabel().should('have.text', watchMultisigs['multisig-with-pure'].name)
+    })
+  })
+
+  it('can see the identicon and badge displayed for a watched multisig', () => {
```

Comment:
I feel like this test could be put together with the one before. Because 90% is the same.